examples/demo/advanced/scalar_image_function_inspector.py

```diff
@@ -101,7 +101,7 @@ def compute_model(self):
             self.model_changed = True
             self._function = self.function
         except:
```

Comment:
While we're at it, do you want to turn this bare `except` into something more reasonable, like `except Exception`?

Reply:
done. i cleaned up the try/except clause into a try/except/else block which is more specific on what can break and how to communicate the fact that something is broken. i've also moved things out of the try block to make it more specific.